build/docker/Datadog.Trace.ClrProfiler.IntegrationTests.sh

```diff
@@ -34,4 +34,4 @@ wait-for-it elasticsearch5:9200 -- \
 wait-for-it sqlserver:1433 -- \
 wait-for-it mongo:27017 -- \
 wait-for-it postgres:5432 -- \
-dotnet vstest test/Datadog.Trace.ClrProfiler.IntegrationTests/bin/$buildConfiguration/$publishTargetFramework/publish/Datadog.Trace.ClrProfiler.IntegrationTests.dll --logger:trx --ResultsDirectory:test/Datadog.Trace.ClrProfiler.IntegrationTests/results
+dotnet vstest test/Datadog.Trace.ClrProfiler.IntegrationTests/bin/$buildConfiguration/$publishTargetFramework/publish/Datadog.Trace.ClrProfiler.IntegrationTests.dll --logger:trx --ResultsDirectory:test/Datadog.Trace.ClrProfiler.IntegrationTests/results --TestCaseFilter:Category!=LinuxUnsupported
```

Comment:
I know you added the `ArmUnsupported` trait to the test to cover this, but I think we should probably add the `LinuxUnsupported` to the arm64 bash script too. e.g. down the line we might support Windows on Arm. Not critical, just a thought